### ISSUE TITLE
How can I disable this completion ?

### ISSUE THREAD

**Issue by rjosodtssp** [dbcli/mycli#617]
![default](https://user-images.githubusercontent.com/11205685/42142042-c07cf65c-7ddf-11e8-9ba8-ea95ab28abe4.png)

I only type `sel`, then it shows up last `select` command I input before, I don't need the whole command, how can I disable it ?

**Comment by amjith** [comment]
:cry: 

Right now there is no easy way to disable it. I'd like to understand why you would like to disable it. Are you finding the auto-suggestion distracting?

Would it help if you chose a different colorscheme that doesn't have the auto-suggestion quite so bright? If you want to try that you can edit your config file located at ~/.myclirc and look for the option called `syntax_style`. 

The comment above that option has a list of all the syntax colorschemes available. 

But if you have a legitimate reason to turn it off, I'm happy to look into it.

**Comment by rjosodtssp** [comment]
Yes, I feel it a little distracting, and I know how to change syntax style, thanks anyway.
